source/getting-started/installation-raspberry-pi-all-in-one.markdown

```diff
@@ -84,6 +84,9 @@ To launch the OZWCP web application:
   Don't check the USB box regardless of using a USB based device.
 </p>
 
+### {% linkable_title Using the GPIOs %}
```

Comment:
@CCOSTAN Make changes to your own branch and it will update your pull request.
In this case edit [this](https://github.com/CCOSTAN/home-assistant.io/blob/patch-16/source/getting-started/installation-raspberry-pi-all-in-one.markdown) file in this [branch](https://github.com/CCOSTAN/home-assistant.io/tree/patch-16). :)